### PR TITLE
Update breakpad v2022.07.12 and use system breakpad on linux

### PR DESCRIPTION
**Pull request by TheOneRing**

### CODE REVIEW

**Comment by TheOneRing** [comment]
Compiles on windows and mac.
For linux we might want to use a system installation of breakpad.
For linux a ./configure script is available which allows building the client lib.

**Comment by TheOneRing** [comment]
@fmoc

File: CMakeLists.txt [hunk 1]
```diff
@@ -12,6 +12,14 @@ endif()
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+if (UNIX AND NOT APPLE)
```
Comment (fmoc) [inline CMakeLists.txt:15]: Please document the backgrounds of this logic. Is there a reason a "system breakpad" can't be used on non-Linux systems and vice versa?

**Review by fmoc — CHANGES_REQUESTED** [review]

**Comment by dschmidt** [comment]
If it works for you, fine

**Pull request merged**